Dwarf.h

```diff
@@ -1,7 +1,22 @@
 #pragma once
 
+#include <string>
+#include "Character.h"
+
 struct Dwarf : Character
 {
-private:
-    const std::string name;
-};
\ No newline at end of file
+    Dwarf
+    (
+        std::string name_, 
+        int hitPoints_, 
+        int armor_
+    );
+
+    const std::string& getName() override;
+
+    std::string getStats() override;
+
+    private:
+
+        const std::string name;
+};
```

Comment:
Revise.  See all of my previous comments about indentation, the way constructors should look, etc..
- [ ] REVISE